commitlintplugins.test.ts

```diff
@@ -212,6 +212,34 @@ test('footer-notes-misplacement-3', () => {
 })
 
 
+test('footer-references-existence1', () => {
+    let commmitMsgwithCorrectFooter = "foo: this is only a title"
+        + "\n\n"+ "Bla bla blah[1]."
+        + "\n\n" + "[1] http://foo.bar/baz"
+    let footerReferenceExistence1 = runCommitLintOnMsg(commmitMsgwithCorrectFooter);
+    expect(footerReferenceExistence1.status).toBe(0);
+})
+
+
+test('footer-references-existence2', () => {
+    let commmitMsgwithWrongFooter = "foo: this is only a title"
+        + "\n\n"+ "Bla bla blah."
+        + "\n\n" + "[1] http://foo.bar/baz"
+    let footerReferenceExistence2 = runCommitLintOnMsg(commmitMsgwithWrongFooter);
+    expect(footerReferenceExistence2.status).not.toBe(0);
+})
+
+
+test('footer-references-existence3', () => {
+    let commmitMsgwithWrongFooter = "foo: this is only a title"
+        + "\n\n"+ "Bla bla blah[1], and after that [2], then [3]."
```

Comment:
@parhamsaremi let's add one more test, in which [4] is used, but there's no footer note for it.

Reply:
I didn't get this. don't we already have [2] which is used in body but there is no footer for it? :thinking:

Reply:
oh true, ok, then let's change this test, I prefer the number not being used being the last one, not the one in the middle